lib/launcher.rb

```diff
@@ -50,7 +50,7 @@ def http_app(http_port, plugins = http_plugins)
         :DoNotListen => true,
         :Port => http_port, # only being used to correctly log http port being used
         :Logger => ::Proxy::LogBuffer::Decorator.instance,
-        :ServerSoftware => '',
+        :ServerSoftware => "foreman-proxy/#{Proxy::VERSION}",
```

Comment:
It's common security practice to not publish the software versions. Do we need this?

Reply:
Ok, fair enough.

Reply:
I would like to see it as a separate commit. It's not obvious that this is part of the httpboot and I wouldn't look at this commit in git log if I was looking for the change.